src/qutip_qip/family.py

```diff
@@ -0,0 +1,9 @@
+"""QuTiP family package entry point."""
+
+
+def about():
+    """Return information to include in qutip.about()."""
+    return "qutip-qip", [
+        "Moo: 1.5",
+        "Bar: 2.0",
```

Comment:
@BoxiLi What information should I include here?

Reply:
Thanks @hodgestar !
Those are the dependent packages I assume? If we exclude those already listed in qutip, we only have optional packages such as `qutip-qtrl` and `qiskit` here.

Reply:
I change the implementation to provide just the version of the family package. We can add more complex functionality later if it turns out we need it.